core/config/init_args.go

```diff
@@ -0,0 +1,199 @@
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/anoideaopen/foundation/proto"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+// FromInitArgs parses positional initialization arguments and generates JSON-config of []byte type.
+// Accepts the channel name (chaincode) and the list of positional initialization parameters.
+// Only needed to maintain backward compatibility.
+// Marked for deletion after all deploy tools will be switched to JSON-config initialization of chaincodes.
+// Deprecated
+func FromInitArgs(channel string, args []string) ([]byte, error) {
+	const minArgsCount = 2
+	argsCount := len(args)
+	if argsCount < minArgsCount {
+		return nil, fmt.Errorf("minimum required args length is '%d', passed %d",
+			argsCount, minArgsCount)
+	}
+
+	var (
+		cfg *proto.Config
+		err error
+	)
+
+	symbol := strings.ToUpper(channel)
+
+	switch channel {
+	case "nft", "dcdac", "ndm", "rub", "it":
```

Comment:
suggest to totaly remove this part, all tokens based on foundation wil configure their expectations about config

Reply:
not yet.